libraries/plugins/transaction_status/transaction_status_plugin.cpp

```diff
@@ -93,8 +235,9 @@ transaction_status_plugin::~transaction_status_plugin() {}
 void transaction_status_plugin::set_program_options( boost::program_options::options_description& cli, boost::program_options::options_description& cfg )
 {
    cfg.add_options()
-      ( TRANSACTION_STATUS_BLOCK_DEPTH_KEY, boost::program_options::value<uint32_t>()->default_value( TRANSACTION_STATUS_DEFAULT_BLOCK_DEPTH ), "Defines the number of blocks from the head block that transaction statuses will be tracked." )
-      ( TRANSACTION_STATUS_TRACK_AFTER_KEY, boost::program_options::value<uint32_t>()->default_value( TRANSACTION_STATUS_DEFAULT_TRACK_AFTER ), "Defines the block number the transaction status plugin will begin tracking." )
+      ( TRANSACTION_STATUS_BLOCK_DEPTH_KEY,   boost::program_options::value<uint32_t>()->default_value( TRANSACTION_STATUS_DEFAULT_BLOCK_DEPTH ), "Defines the number of blocks from the head block that transaction statuses will be tracked." )
+      ( TRANSACTION_STATUS_TRACK_AFTER_KEY,   boost::program_options::value<uint32_t>()->default_value( TRANSACTION_STATUS_DEFAULT_TRACK_AFTER ), "Defines the block number the transaction status plugin will begin tracking." )
+      ( TRANSACTION_STATUS_REBUILD_STATE_KEY, boost::program_options::bool_switch()->default_value( false ),                                      "Indicates that the transaction status plugin must re-build its state upon startup." )
```

Comment:
Should follow pattern as `replay_blockchain` being only a CLI arg (https://github.com/steemit/steem/blob/295aa99d2bbdc9dd1ca35e7b59e2d192307c43ad/libraries/plugins/chain/chain_plugin.cpp#L317).

Or `enable-stale-production` (https://github.com/steemit/steem/blob/295aa99d2bbdc9dd1ca35e7b59e2d192307c43ad/libraries/plugins/witness/witness_plugin.cpp#L437-L448).